README.md

```diff
@@ -1 +1,230 @@
+
+You can build the Ballerina service using `$ ballerina build uuid_service.bal`. You should be able to see the following output. 
+
+```bash
+$ ballerina build uuid_service.bal
+Compiling source
+    uuid_service.bal
+
+Generating executable
+    uuid_service.balx
+	@kubernetes:Service 			 - complete 1/1
+	@kubernetes:Deployment 			 - complete 1/1
+	@kubernetes:Docker 			 - complete 3/3 
+	@kubernetes:Helm 			 - complete 1/1
+
+	Run the following command to deploy the Kubernetes artifacts: 
+	kubectl apply -f /home/laf/dev/uuid-service/kubernetes/
+
+	Run the following command to install the application using Helm: 
+	helm install --name uuid-service-deployment /home/laf/dev/uuid-service/kubernetes/uuid-service-deployment
+```
+
+After the build is done, the docker image would have been created, and pushed to Docker Hub, and also the Kubernetes deployment artifacts would be generated as well.
+    
+## Deployment
+
+- Configuring AKS environment
+
+Before deploying the service on AKS, you will need to setup the AKS environment to create the Kubernetes cluster and deploy an application.
+
+Let's start by installing the Azure command line utilities (CLI) in our local machine. Please refer to [Azure CLI Installation](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) in finding the steps for the installation.
+
+Next step is creating an Azure Service Principal:
+
+```bash 
+$ az ad sp create-for-rbac --skip-assignment
+```
+
+The output would be similar to the following:
+
+```bash 
+{
+  "appId": "e7596ae3-6864-4cb8-94fc-20164b1588a9",
+  "displayName": "azure-cli-2018-10-29-21-37-08",
+  "name": "http://azure-cli-2018-10-29-21-37-08",
+  "password": "52c95f25-bd1e-4314-bd31-d8112b293521",
+  "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db48"
+}
+```
+
+- Create the Kubernetes cluster
+
+```bash 
+$ az aks create \
+    --resource-group myResourceGroup \
+    --name myAKSCluster \
+    --node-count 1 \
+    --service-principal <appId> \
+    --client-secret <password> \
+    --generate-ssh-keys
+```
+
+- Configure kubectl to connect to AKS Kubernetes cluster
+
+The following command will get the credentials for a given Kubernetes cluster in AKS.
+
+```bash
+$ az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
+```
+
+To verify the connection to your cluster, run "kubectl get nodes" command:
+
+```bash
+$ kubectl get nodes
+NAME                       STATUS    ROLES     AGE       VERSION
+aks-nodepool1-29389471-0   Ready     agent     10d       v1.9.11
+
+```
+
+- Deploying the Ballerina service to AKS
+
+Since the Kubernetes artifacts has been automatically built in the earlier Ballerina application build, we simply have to run the following command to deploy the Ballerina service in AKS:
```

Comment:
Change to "Since the Kubernetes artifacts were automatically built in the earlier Ballerina application build..."

Reply:
Fixed, thanks!.